spec/bundler/spec_helper.rb

```diff
@@ -111,16 +110,6 @@ def self.ruby=(ruby)
     end
   end
 
-  config.around :each do |example|
-    Dir.mktmpdir("bundler_commands_console") do |dir|
-      xdg_config_home_backup = ENV.delete("XDG_CONFIG_HOME")
-      ENV["XDG_CONFIG_HOME"] = dir
-      example.run
-    ensure
-      ENV["XDG_CONFIG_HOME"] = xdg_config_home_backup
-    end
-  end
-
```

Comment:
This should be proposed upstream so that syncing does not revert it. I think to be consistent with how we approach this for other similar variables, it could be set to `nil` in the `before(:suite)` block instead.

Reply:
I proposed my suggestion upstream: https://github.com/rubygems/rubygems/pull/4587.